service/src/test/java/uk/gov/hmcts/reform/fpl/model/common/C2DocumentBundleSerializationTest.java

```diff
@@ -26,8 +31,13 @@
     @Autowired
     private ObjectMapper objectMapper;
 
+    @MockBean
```

Comment:
something smells here,  serialisation does not depends on toggle, you can remove this and all tests will be green